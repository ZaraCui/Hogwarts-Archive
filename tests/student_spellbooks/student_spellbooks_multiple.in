ADD COLLECTION tests/data/books_small.csv
ADD STUDENT Ron Weasley
RENT 100000 1001
RENT 100000 1002
STUDENT SPELLBOOKS 100000
EXIT

