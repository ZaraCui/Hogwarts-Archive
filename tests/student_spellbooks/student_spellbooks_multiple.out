user: 12 spellbooks successfully added.

user: Success.

user: Success.

user: Success.

user: Levitation Charms (Helga Hufflepuff)
Dark Arts Basics (Salazar Slytherin)

user: Ending Archive process.
